operator/pkg/model/translation/envoy_http_connection_manager.go

```diff
@@ -53,6 +53,9 @@ func NewHTTPConnectionManager(name, routeName string, mutationFunc ...HttpConnec
 				},
 			},
 		},
+		InternalAddressConfig: &httpConnectionManagerv3.HttpConnectionManager_InternalAddressConfig{
```

Comment:
is this change already changing the behavior. Or not yet because we still don't set the CIDRRanges in the internal address config which ([according to the docs](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-internaladdressconfig)) still treats the RFC IPs as internal.

> If unset, then RFC1918 / RFC4193 IP addresses will be considered internal.

Have you tested this? If it's changing it might be worth a note in the release notes?

Note: alternatively we could also use the runtime config to keep the old behavior (if we don't want to break): https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.31/v1.31.2

Reply:
yep, at least this way it seems to behave the same as today.